package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I don't disagree with moving package.json out of `assets` directory and into root, but I think this is a bigger change and should be handled in a separate PR.

Reply:
Actually, this says `date-block`... not sure what this is.